cosmo_tester/framework/config.py

```diff
@@ -0,0 +1,277 @@
+import pkg_resources
+import warnings
+import yaml
+
+import os
+import sys
+
+
+class SchemaError(Exception):
+    pass
+
+
+class NotSet(object):
+    def __repr__(self):
+        return 'not set'
+
+
+class Config(object):
+    schema = {}
+    raw_config = {}
+    namespaces = set([None])
```

Comment:
They are updated when the config is loaded.

Reply:
yes, and if there could be multiple instances of the Config class, they would all share these, because  those are class attrs

I meant "static" in the C sense, as in, global (but these aren't technically part of the global namespace), I didn't mean "not changing"

I mean this:
```python
>>> class A(object):
...  x = {}
... 
>>> a, b = A(), A()
>>> a.x[1] = 2
>>> b.x
{1: 2}
>>> 
```

I'm actually surprised that the linter didn't say anything about this, I'm pretty sure it would say something about eg. a mutable function default?

Reply:
OK, will shove them into the __init__